neural_mfg.py

```diff
@@ -162,19 +164,92 @@ def sample_room(self):
         X_b = tf.concat([x_b, y_b], axis=1)
 
         return pd.DataFrame(X_b.numpy()), pd.DataFrame(X_in.numpy()), pd.DataFrame(X_out.numpy())
- 
-    def get_loss(self,verbose):
+    
+    def resample(self):
+        '''
+        Residual-based adaptive refinement method with greed (RAR-G).
+        
+        Returns
+        ------- 
+        void - it changes the attributes X_b, X_in, X_out of self
+
+        '''
+        X_b, X_in, X_out = self.sample_room()
+        #all_pts = (pd.concat([X_out,X_in,X_b],axis = 0))
+        n_b = self.N_b
+        n_in = X_in.shape[0]
+        n_out = X_out.shape[0]
+        
+        res_HJB, res_KFP, _, _, _, _ = self.get_loss_terms(0,X_out,X_in,X_b)
+        
+        PDEs_residual = pd.DataFrame(res_HJB.numpy() + res_KFP.numpy())
+        PDEs_residual.sort_values(by=PDEs_residual.columns[1])
```

Comment:
Should you sort in descending order ?

Reply:
I've committed the change, now the residuals are sorted in descending order